### PR TITLE
Documentation changes, fixes #533

### LINKED ISSUE #533: Ensure markdown formatting for inline samples in docs

### PR DESCRIPTION
**Pull request by AaronCritchley**
Chaning non-formatted HTML and CSS to use the correct md formatting. Fixes [issue #533](https://github.com/ampproject/amphtml/issues/533).

### CODE REVIEW

**Comment by erwinmombay** [comment]
can you de-indent this? other than that, looks good! will LGTM once you are done.

**Comment by AaronCritchley** [comment]
Absolutely, what's preferred? 
Indentation in line with the code or one block behind?

**Comment by erwinmombay** [comment]
lets just go ahead and follow the main `html` sample on https://raw.githubusercontent.com/ampproject/amphtml/master/README.md (where is its aligned-left with no indent unless nesting of code/markup of course)

**Comment by AaronCritchley** [comment]
Should be better, the general Google consensus seems to be 2 spaces so I tried to stick to that. If you would prefer otherwise, please let me know and I'll be happy to change!

**Comment by erwinmombay** [comment]
@AaronCritchley nope, looks great!

**Comment by erwinmombay** [comment]
actually, just rechecked. if its a nesting of elements, it should be 2 spaces.

```html
<div>
  <div></div>
</div>
```

if you are breaking on a property, it should be 4 spaces.

```html
<div class=b
    placeholder
    data-something=value>
</div>
```

i hope you don't mind. sorry bout that!

**Comment by AaronCritchley** [comment]
Sure, I'll go through and change them, no worries! 
Just to confirm, on your second example, do you prefer for the property to be broken on the same line or on the next? e.g.

```html
<div class=b
    placeholder
    data-something=value>
</div>
```
OR
```html
<div class=b
    placeholder
    data-something=value
    >
</div>
```

**Comment by erwinmombay** [comment]
i believe we prefer the former.

```html
<div class=b
    placeholder
    data-something=value>
</div>
```

**Comment by AaronCritchley** [comment]
Done! 
Is there anywhere public that style preferences are documented so I can get it right first time in the future?

**Comment by erwinmombay** [comment]
we're sort of loosely basing this on the [google js style guide](https://google.github.io/styleguide/javascriptguide.xml) were if the current line is a continuation of the previous line then indent it by 4. haha granted we haven't been very good at being strict about this either :D so we really appreciate this sort of work to fix it.

File: docs/create_page.md [hunk 1]
```diff
@@ -8,31 +8,31 @@ stage it, test it, and get it ready for publication.
...
-      </body>
-    </html>
+<!doctype html>
+<html amp lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Hello, AMPs</title>
+    <link rel="canonical" href="http://example.ampproject.org/article-metadata.html" />
+    <meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1,user-scalable=no,minimal-ui">
+      <script type="application/ld+json">
```
Comment (erwinmombay) [inline docs/create_page.md]: can you de-indent the script here by 2 to be inline with `meta`.

File: docs/create_page.md [hunk 2]
```diff
@@ -60,33 +60,32 @@ To get you started,
...
-      </body>
-    </html>
+<!doctype html>
+<html AMP lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Hello, AMPs</title>
+    <link rel="canonical" href="http://example.ampproject.org/article-metadata.html" />
+    <meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1,user-scalable=no,minimal-ui">
+      <script type="application/ld+json">
```
Comment (erwinmombay) [inline docs/create_page.md]: ditto on this one.

File: docs/create_page.md [hunk 3]
```diff
@@ -101,42 +100,41 @@ using class or element selectors in an author-defined,
...
```
Comment (erwinmombay) [inline docs/create_page.md]: ditto.

**Comment by erwinmombay** [comment]
just the final change to those script tags and ill merge :dancer:

**Comment by AaronCritchley** [comment]
Done! 
If you ever want me to tidy up stuff like this again feel free to tag me and I'll happily complete it, happy to help wherever I can :+1:

**Comment by erwinmombay** [comment]
@AaronCritchley really appreciate that :+1:

**Comment by erwinmombay** [comment]
Merged!

**Pull request merged**